dear_petition/petition/export/mapper.py

```diff
@@ -44,157 +49,34 @@ def map_petitioner(data, petition, extra={}):
 
 
 def map_attorney(data, petition, extra={}):
-    pass
+    attorney = extra.get("attorney", None)
+    data["NameAtty"] = getattr(attorney, "name", None)
+    data["StAddrAtty"] = getattr(attorney, "address1", None)
+    data["MailAddrAtty"] = getattr(attorney, "address2", None)
+    data["CityAtty"] = getattr(attorney, "city", None)
+    data["StateAtty"] = getattr(attorney, "state", None)
+    data["ZipCodeAtty"] = getattr(attorney, "zipcode", None)
+    ##### petition to expunge section ########
+    # This section is signed by the Attorney
+    ###########################################
+    data["PetitionNotFiledSignName"] = getattr(attorney, "name", None)
+    data["PetitionerAttorneyCbx"] = Checkbox("Yes")
+    data["PetitionNotFiledSignDate"] = dt_obj_to_date(dt.datetime.today()).strftime(
+        constants.DATE_FORMAT
+    )
 
 
 def map_agencies(data, petition, extra={}):
-    pass
+    agencies = extra.get("agencies", {})
```

Comment:
I think agencies is guaranteed to be a list from the serializer, so I think we can just iterate over these, e.g.:

```python
for idx, agency in enumerate(agencies):
    data[f"NameAgency{idx}"] = agency.name
    # ...
```

Reply:
done and updated test